_data/testingcodeswanlake.yml

```
@@ -31,28 +31,28 @@
     - title: Configuring Tests
       url : '#'
       sublinks:
-        - title: Setup and Teardown
-          url: /learn/testing-ballerina-code/configuring-tests/#setup-and-teardown
-          active: setup-and-teardown
-        - title: Test Specific Configurations
-          url: /learn/testing-ballerina-code/configuring-tests/#test-specific-configurations
-          active: test-specific-configurations
-        - title: Test-only Dependencies
-          url: /learn/testing-ballerina-code/configuring-tests/#test-only-dependencies
-          active: test-only-dependencies
+        - title: Setting Up and Tearing Down
+          url: /learn/testing-ballerina-code/configuring-tests/#settingup-and-tearingdown
+          active: settingup-and-tearingdown
+        - title: Defining Test Specific Configurations
+          url: /learn/testing-ballerina-code/configuring-tests/#defining-test-specific-configurations
+          active: defining-test-specific-configurations
+        - title: Defining Test-only Dependencies
+          url: /learn/testing-ballerina-code/configuring-tests/#defining-test-only-dependencies
+          active: defining-test-only-dependencies
     - title: Code Coverage and Reporting
       url : '#'
       sublinks:
-        - title: Test Report
-          url: /learn/testing-ballerina-code/code-coverage-and-reporting/#test-report
-          active: test-report
-        - title: Code Coverage Report
-          url: /learn/testing-ballerina-code/code-coverage-and-reporting/#code-coverage-report
-          active: code-coverage-report
-        - title: JaCoCo XML Report
-          url: /learn/testing-ballerina-code/code-coverage-and-reporting/#jacoco-xml-report
-          active: jacoco-xml-report
-    - title: Data-Driven Tests
+        - title: Generating Test Report
```

```suggestion
        - title: Generating Test Report
```
Should be `Generating a/the Test Report`.